### ISSUE TITLE
ENTRYPOINT can make a docker image unusable in CWL

### ISSUE THREAD

**Issue by bosonogi** [rabix/bunny#251]
If a docker image defines an entrypoint, the command line that bunny generates (`/bin/sh -c ...`) most likely will not execute correctly.

For an example, if the entrypoint is defined to be _python_, what actually winds up executing in the container is `python /bin/sh -c ...` with predictably failing results.

**Comment by milos-ljubinkovic** [comment]
Entrypoint handling has been merged in release v1.0.2

**Comment by aurelienmazurie-wk** [comment]
I'm not sure if this is fixed, or maybe not completely. My use case is a Docker container with an `ENTRYPOINT` together with a `WORKDIR` command to run a Python script:

```Dockerfile
WORKDIR /app
COPY test.py /app
ENTRYPOINT ["python", "test.py"]
```

This works fine on the command line (e.g., `docker run <image>`) but fail with Rabix with the following error message: `python: can't open file 'test.py': [Errno 2] No such file or directory`.

Would it be possible that CWL modifies the `WORKDIR`? I know `cwltool` does.
